visidata/sheets.py

```diff
@@ -954,7 +954,7 @@ def quit(vd, *sheets):
 globalCommand('S', 'sheets-stack', 'vd.push(vd.sheetsSheet)', 'open Sheets Stack: join or jump between the active sheets on the current stack')
 globalCommand('gS', 'sheets-all', 'vd.push(vd.allSheetsSheet)', 'open Sheets Sheet: join or jump between all sheets from current session')
 
-BaseSheet.addCommand('^R', 'reload-sheet', 'reload(); recalc(); status("reloaded")', 'reload current sheet'),
+BaseSheet.addCommand('^R', 'reload-sheet', 'reload(); _dm_reset(); recalc(); status("reloaded")', 'reload current sheet'),
```

Comment:
Every sheet now does the `_dm_reset()` upon reload, regardless if `defer=True` or not. I also feel uncomfortable with this.